workflow-templates/aws-ecr-token.properties.json

```
@@ -0,0 +1,5 @@
+{
+    "name": "Pull AWS ECR image for use",
```

```suggestion
    "name": "Pull AWS ECR Image",
```

suggesting this change as I think it would look better in the UI